tests/FakeItEasy.Specs/UnnaturalFakeSpecs.cs

```diff
@@ -197,6 +198,59 @@ public static void CallsToNonVoidWithCallToWrongFake(Fake<IFoo> fake, IFoo wrong
         }
 
         [Scenario]
+        [Example(int.MinValue)]
+        [Example(-42)]
+        [Example(0)]
+        [Example(42)]
+        [Example(int.MaxValue)]
+        public static void CallsToSetAnyValue(int value, Fake<IFoo> fake, bool wasCalled)
+        {
+            "Given an unnatural fake"
+                .x(() => fake = new Fake<IFoo>());
+
+            "And assignment of the property is configured for any value"

```

Comment:
nit: "the" suggests an antecedent, which we don't have. "a property" is probably better